src/ORM/FieldType/DBField.php

```diff
@@ -603,40 +520,31 @@ public function debug()
 DBG;
     }
 
-    public function __toString()
+    public function __toString(): string
     {
         return (string)$this->forTemplate();
     }
 
-    /**
-     * @return array
-     */
     public function getArrayValue()
     {
         return $this->arrayValue;
     }
 
-    /**
-     * @param array $value
-     * @return $this
-     */
-    public function setArrayValue($value)
+    public function setArrayValue($value): static
     {
         $this->arrayValue = $value;
         return $this;
     }
 
     /**
-     * Get formfield schema value
-     *
-     * @return string|array Encoded string for use in formschema response
+     * Get formfield schema value for use in formschema response
      */
-    public function getSchemaValue()
+    public function getSchemaValue(): mixed
     {
         return $this->RAW();
```

Comment:
The raw value could be anything, so we can't constrain this type to `string|array` despite what the PHPDoc says.